app/models/game.rb

```diff
@@ -17,8 +27,23 @@ def populate_board!
     # coordinates as you normally might for a matrix)
     populate_white_pieces
     populate_black_pieces
+    white_user_id.nil? ? set_black_player_id : set_white_player_id
```

Comment:
sets appropriate white/black id for current user depending on if they clicked the checkbox in the creation form